src/common-lib/error-reporter/errorReporter.ts

```diff
@@ -208,6 +208,11 @@ const errorReporter = (
         return;
       }
 
+      if (maybeError instanceof OakError && !maybeError.config.shouldNotify) {
+        logger.log("Error should not be reported, aborting reportError()");
+        return;
```

Comment:
I'd _err_ towards logging out the actual error here too?
```suggestion
        logger.log("Error should not be reported, aborting reportError()");
        logger.error(maybeError);
        return;
```